internal/wrappers/feature-flags-http.go

```diff
@@ -64,3 +65,49 @@ func (f FeatureFlagsHTTPWrapper) GetAll() (*FeatureFlagsResponseModel, error) {
 		return nil, errors.New("failed to load feature flags for tenant")
 	}
 }
+
+func (f FeatureFlagsHTTPWrapper) GetSpecificFlag(flagName string) (*FeatureFlagResponseModel, error) {
+	accessToken, tokenError := GetAccessToken()
+	if tokenError != nil {
+		return nil, tokenError
+	}
+
+	tenantIDFromClaims, extractError := extractFromTokenClaims(accessToken, tenantIDClaimKey)
+	if extractError != nil {
+		return nil, extractError
+	}
+
+	tenantID := strings.Split(tenantIDFromClaims, "::")[1]
```

Comment:
please extract this common logic to a private method (extracting access token and tenant)

Reply:
Done